### DISCUSSION TITLE
Proof Params Init with Error and Warn?

### DISCUSSION THREAD

**Discussion by lingjianrui** [filecoin-project/lotus#7024] [General Q&A] [answered]
Hi Guys,

I Followed the guide and setup Env Variables.  
export BELLMAN_CPU_UTILIZATION=0.875
export LOTUS_MINER_PATH=/big_data/filecoin_data/.lotusminer
export LOTUS_PATH=/big_data/filecoin_data/.lotus
export FIL_PROOFS_PARAMETER=/big_data/filecoin_data/proof
export FIL_PROOFS_PARENT_CACHE=/big_data/filecoin_data/proof_parent_cache

But ...  There is a lot of warning tips  while execute  lotus daemon.
please check below picture.
![image](https://user-images.githubusercontent.com/44353652/128957997-7d021e33-9188-40b3-986e-857bf8839b07.png)

Is this progress perform normal ? 
Any help would be appreciate.
Thanks in advance.

**Comment by de-authority** [comment, marked as answer]
I think that's normal. does your lotus sync stucked? if not. ignore  those warning.

**Reply by LexLuthr** [reply]
@lingjianrui hey, do you still need help with this? Or does the above suggestion works for you?
